src/Shift-Changes.package/ShAbstractChangeDetector.class/README.md

```diff
@@ -0,0 +1,8 @@
+I detect the changes to a given Class.
+My subclasses implements 
+
+#compareClass: oldClass with: aBuilder to calculate the changes performed to the class.
```

Comment:
the comment seems to be old because this method is implemented only in this abstract class